database/migrations/2022_01_25_203000_add_dework_organization_id.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddDeworkOrganizationId extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('circles', function (Blueprint $table) {
+            $table->string('dework_organization_id')->nullable();
```

Comment:
I think a separate table with many-to-one support for external integratinos would be a more reusable model. Allowing for any number of external integrations.

Reply:
idk, if it is only one field...

I think it would attach to the `protocol` table

Is this id secret at all? if not, easier as we don't have to think about permissions for it.

And then, I wonder how the permissions work once in coordinape, the user needs an access token for both front ends.. hm

And like Zashton noted in coordinape discord:
This also needs to happen in hasura, as that is handling our migrations now